README.md

```diff
@@ -14,6 +14,8 @@ publishing, and versioning guidance.
 If you are new, there's also a simpler introduction in the
 [Hello world JavaScript action repository](https://github.com/actions/hello-world-javascript-action).
 
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/actions/typescript-action)
```

Comment:
In most cases, a user would want to clone the template first, then open a codespace under their own org/user.

What do you think about moving this button to after the "1. Clone your new repository" step?

Reply:
The main issue is that the button wouldn't change to point to their repo, which is a nuisance. I'm also thinking just removing the button makes sense.

Reply:
Removed the button :)